docs/system-admin-guide/custom-fields/README.md

```diff
@@ -30,21 +30,24 @@ Depending on the module, for which the new custom field is being  created, sligh
 1. Custom field **Name**, which will appear as the attribute name on the work package form.
 2. Choose a **Format** of the custom field, e.g. text, list, date, boolean. You can choose between the following formats for custom fields:
 
-![custom fields format](image-20200122112806127.png)
+![Custom fields formats in OpenProject](openproject_system_guide_new_custom_field_formats.png)
+
+> [!TIP]
+> Depending on the format of the custom field, the fields to specify vary.
+4. Specify the **minimal and maximal length**. If a 0 is chosen, no restriction will be imposed on the length of the custom field.
```

Comment:
I think, there is the mention of `multi-select` missing, isn't it?

I mean, it is explained later, but why the difference?

Reply:
@Kharonus here on the screenshot we have the type *text*, so multi-select is not displayed and it would be odd to mention it here.